unicodetools/src/main/java/org/unicode/text/utility/Settings.java

```diff
@@ -23,16 +25,81 @@
     /**
      * Used for the default version.
      */
-    public static final String latestVersion = "14.0.0";
-    public static final String lastVersion = "13.1.0"; // last released version
-    // TODO: there is no Unicode 13.1, see https://github.com/unicode-org/unicodetools/issues/100
+    public static final String latestVersion = "15.0.0";
+    public static final String lastVersion = "14.0.0"; // last released version
+
+    private static final String TRIMMED_LATEST_VERSION = trimVersion(latestVersion);
+
+    // See https://github.com/unicode-org/unicodetools/issues/144
+    public static final String NEXT_VERSION_FOLDER_NAME = "dev";
 
     public static final boolean BUILD_FOR_COMPARE =
             CldrUtility.getProperty("BUILD_FOR_COMPARE", "false").startsWith("t");
 
     public static final boolean SKIP_COPYRIGHT =
             "skip".equalsIgnoreCase(CldrUtility.getProperty("copyright", "skip"));
 
+    /**
+     * Removes one or more trailing ".0" substrings, and/or one trailing "-Update".
+     * For quick comparison of version strings and folder names
+     * without building VersionInfo objects.
+     */
+    private static final String trimVersion(String version) {
+        int length = trimmedVersionLength(version);
+        return length == version.length() ? version : version.substring(0, length);
+    }
+
+    private static final int trimmedVersionLength(String version) {
+        int length = version.length();
+        if (version.endsWith("-Update")) {
+            length -= "-Update".length();
+        }
+        while (length >= 2 && version.charAt(length - 2) == '.' && version.charAt(length - 1) == '0') {
+            length -= 2;
+        }
+        return length;
+    }
+
+    /** Ignores trailing ".0" and "-Update". */
+    private static final boolean isLatestVersion(String version) {
+        int length = trimmedVersionLength(version);
+        return length == TRIMMED_LATEST_VERSION.length() &&
+                TRIMMED_LATEST_VERSION.regionMatches(0, version, 0, length);
+    }
+
+    private static final Path getPath(Path parentPath, String relativeDir, String version) {
```

Comment:
nit: probably document that it can handle the fact that the latest version may be in a `dev/` folder

Reply:
Diffs: https://github.com/unicode-org/unicodetools/compare/5a22da5..71ed1d4

Reply:
In ICU4X we basically don't amend/rebase unless there are conflicts and we squash merge, but I do prefer being able to maintain a logical commit history that gets checked in, especially since most PRs to this repo will contain data changes that we want to be split up.

So using the bot is probably a good idea.